enterprise/views.py

```diff
@@ -705,6 +705,26 @@ def get_final_price(self, mode, request):
             )
         return mode['original_price']
 
+    @staticmethod
+    def post_order_to_ecommerce(request, sku):
+        """
+        Post an order for audit enrollment  for the user.
+        """
+        try:
+            api = ecommerce_api_client(request.user)
+            response_data = api.baskets.post({
+                'products': [{'sku': sku}],
+                'checkout': True,
+            })
+        except HttpClientError:
+            logger.error(
+                "Failed to post audit enrollment for product '{sku}'".format(
```

Comment:
If there is an error we should also log the user id so that an enrollment record could be created for the learner after the fact.